modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/NumericDDMFormFieldTypeUtilTest.java

```diff
@@ -0,0 +1,162 @@
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * @author Carolina Barbosa
+ */
+@RunWith(PowerMockRunner.class)
+public class NumericDDMFormFieldTypeUtilTest
+	extends BaseDDMFormFieldTypeSettingsTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_setUpLanguageUtil();
+	}
+
+	@Test
+	public void testGetInputMaskParametersDouble() {
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty("inputMask", true);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"double", ddmFormField, new DDMFormFieldRenderingContext());
+
+		Assert.assertEquals(parameters.toString(), 6, parameters.size());
+		Assert.assertTrue(parameters.containsKey("append"));
+		Assert.assertTrue(parameters.containsKey("appendType"));
+		Assert.assertTrue(parameters.containsKey("decimalPlaces"));
+		Assert.assertTrue(parameters.containsKey("inputMask"));
+		Assert.assertTrue(parameters.containsKey("numericInputMask"));
+		Assert.assertTrue(parameters.containsKey("symbols"));
+	}
+
+	@Test
+	public void testGetInputMaskParametersInteger() {
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty("inputMask", true);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"integer", ddmFormField, new DDMFormFieldRenderingContext());
+
+		Assert.assertEquals(parameters.toString(), 2, parameters.size());
+		Assert.assertTrue(parameters.containsKey("inputMask"));
+		Assert.assertTrue(parameters.containsKey("inputMaskFormat"));
+	}
+
+	@Test
+	public void testGetParametersDouble() {
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
+			new DDMFormFieldRenderingContext();
+
+		ddmFormFieldRenderingContext.setLocale(LocaleUtil.US);
+
+		Map<String, Object> parameters =
+			NumericDDMFormFieldTypeUtil.getParameters(
+				"double", new DDMFormField("field", "numeric"),
+				ddmFormFieldRenderingContext);
+
+		Assert.assertEquals(parameters.toString(), 2, parameters.size());
+		Assert.assertEquals(
+			HashMapBuilder.put(
+				"en_US",
+				HashMapBuilder.put(
+					"decimalSymbol", "."
+				).put(
+					"thousandsSeparator", ","
+				).build()
+			).put(
+				"pt_BR",
+				HashMapBuilder.put(
+					"decimalSymbol", ","
+				).put(
+					"thousandsSeparator", "."
+				).build()
+			).build(),
+			parameters.get("localizedSymbols"));
+		Assert.assertEquals(
+			HashMapBuilder.put(
+				"decimalSymbol", "."
+			).put(
+				"thousandsSeparator", ","
+			).build(),
+			parameters.get("symbols"));
+	}
+
+	@Test
+	public void testGetParametersInteger() {
+		Map<String, Object> parameters =
```

Comment:
Shouldn't maps variables have the suffix `Maps`? [Reference](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#[inlineExtension]Maps-are-plurals,-with-suffix).